### DISCUSSION TITLE
led-strip

### DISCUSSION THREAD

**Discussion by SergPiven** [UncleRus/esp-idf-lib#306] [Q&A] [answered]
where is the implementation of the function led_strip_install();
   In project (https://github.com/UncleRus/esp-idf-lib/tree/master/examples/led_strip)

**Comment by UncleRus** [comment, marked as answer]
https://github.com/UncleRus/esp-idf-lib/blob/master/components/led_strip/led_strip.c

Perhaps next time it will be easier and faster to use the search.
